enterprise_access/apps/api_client/tests/test_license_manager_client.py

```diff
@@ -38,3 +46,267 @@ def test_get_subscription(self, mock_oauth_client):
             expected_url,
             timeout=settings.LICENSE_MANAGER_CLIENT_TIMEOUT,
         )
+
+
+class TestLicenseManagerUserApiClient(TestCase):
+    """
+     Test License Manager with BaseUserApiClient.
+     """
+    def setUp(self):
+        super().setUp()
+        self.api_base_url = LicenseManagerUserApiClient.api_base_url
+        self.factory = RequestFactory()
+        self.faker = FakerFactory.create()
+
+        self.mock_enterprise_customer_uuid = self.faker.uuid4()  # pylint: disable=no-member
+        self.mock_enterprise_customer_slug = "test_slug"
+        self.mock_enterprise_catalog_uuid = self.faker.uuid4()  # pylint: disable=no-member
+        self.mock_user_email = self.faker.email()  # pylint: disable=no-member
+        self.mock_learner_license_uuid = self.faker.uuid4()  # pylint: disable=no-member
+        self.mock_learner_license_activation_uuid = self.faker.uuid4()  # pylint: disable=no-member
+        self.mock_license_activation_key = self.faker.uuid4()  # pylint: disable=no-member
+        self.mock_auto_apply_uuid = self.faker.uuid4()  # pylint: disable=no-member
+        self.mock_subscription_plan_uuid = self.faker.uuid4()  # pylint: disable=no-member
+        self.mock_customer_agreement_uuid = self.faker.uuid4()  # pylint: disable=no-member
```

Comment:
[nit / sanity check] Are these disabled pylint rules necessary?

Reply:
Yep, otherwise it throw's a linting error.

Reply:
Looks like modifying `self.faker` a bit would resolve the linting error, without disabling any rules.

```python
from faker import Faker
self.faker = Faker()
self.mock_enterprise_customer_uuid = self.faker.uuid4()
```

`make quality` passes without disabling lint.